package.json

```diff
@@ -19,7 +19,7 @@
   "sideEffects": false,
   "scripts": {
     "prepublishOnly": "npm run build",
-    "clean": "rimraf dist",
+    "clean": "rimraf dist && rimraf node_modules/.cache",
```

Comment:
rollup was not seeing my changes to `types.ts` and I had to manually delete this directory to fix it. Figured it'd be good to automate